src/Symfony/Component/Messenger/Command/ConsumeMessagesCommand.php

```diff
@@ -179,7 +179,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         }
 
         if (null !== ($timeLimit = $input->getOption('time-limit'))) {
-            if (!is_numeric($timeLimit) || 0 >= $limit) {
+            if (!is_numeric($timeLimit) || 0 >= $timeLimit) {
```

Comment:
Can you add a test to cover that?

Reply:
See my Pr description, is slow test is acceptable ?

Reply:
Test added